src/outdated-browser-rework.js

```diff
@@ -0,0 +1,5 @@
+import outdatedBrowser from 'outdated-browser-rework';
+import 'outdated-browser-rework/dist/style.css';
+
+// Use the default versions until we have something meaningful to add?
```

Comment:
We might want to follow the PF support. https://www.patternfly.org/get-started/about/#supported-browsers If PF is not working, the UI won't work.

Reply:
if we're moving to just "latest", we should also update https://github.com/RedHatInsights/insights-chrome/blob/d9cba3c94d516a28935796bc79e54a349cb873b8/src/components/Footer/Footer.tsx#L83 so we're not having two different answers